instrumentation/spring/spring-batch-3.0/javaagent/spring-batch-3.0-javaagent.gradle

```diff
@@ -30,7 +30,7 @@ test.finalizedBy(tasks.register("testChunkRootSpan", Test) {
   filter {
     includeTestsMatching '*ChunkRootSpanTest'
   }
-  jvmArgs '-Dotel.instrumentation.spring-batch.chunk.root-span=true'
+  jvmArgs '-Dotel.instrumentation.spring-batch.chunk.new-trace=true'
```

Comment:
Strong +1 to new-trace always much more grokkable than talking about root spans.

Reply:
Should we add `experimental` to this property?